plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryConnectorModule.java

```diff
@@ -78,15 +78,14 @@ public BigQueryClient provideBigQueryClient(BigQueryConfig config, HeaderProvide
     static String calculateBillingProjectId(Optional<String> configParentProjectId, Optional<Credentials> credentials)
```

Comment:
Commit message: 
```
Clean up Optional usage in BigQueryConnectorModule
```
?

`BQCM` could be a bit misleading

Reply:
fixed in amended commit: 1005576e26b052427de07609d1cc7270fcfbd3b8